stories/CatalogHelper.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import CatalogHelper from '../src/helpers/CatalogHelper';
+import {Kpi} from '../src/components/simple/Kpi';
+import '../styles/scss/charts.scss';
+
+import * as catalogJson from './data/catalog.json';
+
+const C = new CatalogHelper(catalogJson as any);
```

Comment:
as `any`?

Reply:
That json import is typed as strange type which is not compatibile with anything, until it's typed as `any`. Typing as ICatalog doesn't work. 
See error below:

`Error:(10, 29) TS2345:Argument of type 'typeof '*.json'' is not assignable to parameter of type 'ICatalog'.
  Property 'metrics' is missing in type 'typeof '*.json''.`